src/hott/11-homotopy-pullbacks.rzk.md

```diff
@@ -411,3 +411,160 @@ from composition and cancelling laws for equivalences.
 
 #end homotopy-cartesian-horizontal-calculus
 ```
+
+## Fiber products
+
+Given two type families `B C : A → U`, we can form their fiberwise product.
```

Comment:
```suggestion
Given two type families `B C : A → U`, we can form their **fiberwise product**.
```
Did you define this in another PR as well?

Reply:
So far I always just added a note saying "depends on...", but maybe you are right that marking as draft might be better practice.